packages/jest-config/src/Deprecated.ts

```diff
@@ -94,6 +94,12 @@ const deprecatedOptions: DeprecatedOptions = {
   )}.
 
   Please update your configuration.`,
+
+  timers: (_options: {timers?: string}) => `  Option ${chalk.bold(
```

Comment:
should we print migration guides? e.g. how to convert `timers: 'real'`, `timers: 'fake'`, `timers: 'modern'` and `timers: 'legacy'` into the new format?